zerver/webhooks/grafana/fixtures/alert_new.json

```diff
@@ -18,7 +18,10 @@
         "silenceURL": "https://zuliptestingwh2.grafana.net/alerting/silence/new?alertmanager=grafana&matcher=alertname%3DTestAlert&matcher=instance%3DGrafana",
         "dashboardURL": "",
         "panelURL": "",
-        "valueString": "[ metric='foo' labels={instance=bar} value=10 ]"
+        "values": {
+          "foo": 10
+        },
+        "imageURL": "https://grafana.com/assets/img/blog/mixed_styles.png"
```

Comment:
So the fact that you're editing the fixture suggests to me that this may be a grafana version-specific format; I worry a little that this change might cause the integration to crash with older versions.

Does grafana have an API changelog that would let us verify what is expected?

Reply:
[This](https://grafana.com/docs/grafana/latest/alerting/alerting-rules/manage-contact-points/integrations/webhook-notifier/#removed-fields-related-to-dashboards) is the documentation I was working off of, but you're right that I didn't go back to previous versions to see what was available in each one. Going back even to the [last major version](https://grafana.com/docs/grafana/v9.5/alerting/manage-notifications/webhook-notifier/) I can see there are some slight differences. How do you suggest I handle this - have slightly different formats for each grafana version?

Reply:
Yeah I think you can just add new fixtures for the latest version, and leave the existing fixtures/tests for the previous version in place to help us make sure we handle them reasonably.

(Did you capture actual fixtures from a Grafana installation for the new one you added?)

Reply:
In 1a4213c, I have reverted the existing fixtures to their previous state and added an additional new one. Fixture `alert_new_values.json` is one that I captured from my own grafana instance, with the only difference being I replaced the URLs to not contain my personal domain name. Fixture `alert_new_no_alertname.json` is an exact copy of `alert_new_values.json` except with the `alertname` field deleted. Without this, I was getting an error in the linter about not 100% test coverage. I haven't witnessed any webhooks that don't have the `alertname` label, but nowhere in the docs does it guarantee it will be there, so I have a fallback.